src/components/goalDetail/GoalHeader.tsx

```
@@ -1,154 +1,261 @@
+import { useGoals, useUpdateGoal, useDeleteGoal } from '@/hooks/useGoals';
+import goalColors from '@/presets/goalColors';
+import colors from '@/presets/colors';
+import ConfirmModal from '../common/ConfirmModal';
+import { useClickOutside } from '@/hooks/useClickOutside';
 
-type Props = {
-  doneItems: number;
-  todoItems: number;
+interface Props {
   id: string;
   setGoalAvailable: (value: boolean) => void;
```

ESLint

[error] 'value' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)